login.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
```

Comment:
O nome do arquivo tem que ser index.html
Senão quando for pra main vamos ter arquivos separados, até podíamos fazer assim e depois alguém juntar tudo, porém a intenção é trabalhar como na vida real, onde alteremos todas a mesma classe.

Reply:
certo , corrigindo o nome